multiplatform-movies/android/src/main/java/com/novoda/movies/gallery/GalleryAdapter.kt

```diff
@@ -0,0 +1,45 @@
+package com.novoda.movies.gallery
+
+import android.support.v7.widget.RecyclerView
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.ImageView
+import com.bumptech.glide.Glide
+import com.novoda.movies.R
+
+
+class GalleryAdapter(private val moviePosterWidthInPixels: Int) : RecyclerView.Adapter<GalleryItemViewHolder>() {
+
+    private var moviePosters = listOf<MoviePoster>()
```

Comment:
Any reason why this is not a `val`?

Reply:
they replace the entire array when the adapter is updated

Reply:
yes but we usually `clear` and `addAll` which avoids creating a new list.
Not a biggy though